parachains/chain-specs/statemine.json

```diff
@@ -14,7 +22,7 @@
     "tokenDecimals": 12,
     "tokenSymbol": "KSM"
   },
-  "relay_chain": "kusama",
+  "relay_chain": "ksmcc3",
```

Comment:
cc @wirednkod

Reply:
> Why this `ksmcc3`?

Ksmcc3 is the ID of the chain. Instead of `Kusama` which is the name we decided to go with IDs cause it helps you identify not only the chain but also the version of the chain that is intended to be used. A good example is rococo. When it was updated from `rococo_v2_1` to `rococo_v2_2` the chainspec changed, but both versions were running. If we were using the `name` instead of the `id` the user would not know (or be able to choose for what its worth) the version that was intended to use.

Reply:
However, doing `ksmcc3` would indeed break things here.
For example, if I try to start a collator with this chainspec, the following happens:
```
$ ./target/release/polkadot-parachain --chain parachains/chain-specs/statemine.json 
2022-06-27 14:49:45 Polkadot parachain    
2022-06-27 14:49:45 ✌️  version 0.9.220-3e8a5465cf8    
2022-06-27 14:49:45 ❤️  by Parity Technologies <admin@parity.io>, 2017-2022    
2022-06-27 14:49:45 📋 Chain specification: Statemine    
2022-06-27 14:49:45 🏷  Node name: serious-pie-1933    
2022-06-27 14:49:45 👤 Role: FULL    
2022-06-27 14:49:45 💾 Database: RocksDb at /home/bear/.local/share/polkadot-parachain/chains/statemine/db/full    
2022-06-27 14:49:45 ⛓  Native runtime: statemine-9220 (statemine-0.tx6.au1)    
Error: Input("Relay chain argument error: Invalid input: Error opening spec file `ksmcc3`: No such file or directory (os error 2)")
```

There's no simple solution here, but we need to have some standardized solution for the entire ecosystem. This comes back to the discussion I started here: https://github.com/paritytech/substrate-connect/issues/1085

The point here is not even just to find a solution for the tutorial, which is just a small effort that is facing the general public with as little friction as possible. The main point is about the strategy we formulate around how the ecosystem will be using chainspecs, where they get them from, and how they look like. We should avoid fragmentation on chainspecs as much as possible.

cc @bkchr @tomaka

Reply:
Please discuss this in https://github.com/paritytech/cumulus/issues/1205